examples/lfric/eg17/full_example_netcdf/Makefile

```diff
@@ -40,7 +40,9 @@
 # export F90=gfortran
 # export F90FLAGS="-Wall -g -fcheck=bound"
 
-include ../../common.mk
+PSYROOT=../../../..
```

Comment:
I have the same Intel compilation problems with this example as with the `full_example` above.

Reply:
As above.